modules/kargoBidAdapter.js

```diff
@@ -13,6 +13,7 @@ const BIDDER = Object.freeze({
   REQUEST_ENDPOINT: '/api/v1/prebid',
   TIMEOUT_ENDPOINT: '/api/v1/event/timeout',
   GVLID: 972,
+  //  NATIVE?? Referenced below as well as https://docs.prebid.org/dev-docs/bidders/kargo.html
```

Comment:
We support native through Kargo Ad Tag so the ad markup submitted would be considered banner, there isn't any news about us supporting native responses directly, so i'm incline to say it's fine to leave out.